src/NoTargets/Sdk/Sdk.targets

```diff
@@ -44,4 +44,8 @@
   </PropertyGroup>
 
   <Import Project="$(CustomAfterNoTargets)" Condition="'$(CustomAfterNoTargets)' != '' and Exists('$(CustomAfterNoTargets)')" />
+  
+  <!-- For CPS/VS support. See https://github.com/dotnet/project-system/blob/master/src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets#L60 -->
```

Comment:
Please move this to before the `CustomAfterNoTargets`

Reply:
makes sense